src/pages/projects.js

```diff
@@ -1,11 +1,18 @@
+import { PageHeader } from '../components/page-header/page-header'
+import { useState } from 'react'
 import { ProjectContent } from '../components/project-content/project-content'
+import { projectsDatas } from '../components/projects-datas'
 export const ProjectsPage = () => {
+    const [datas, setDatas] = useState([{ projectsDatas }])
     return <div>
-        <h1>My projects</h1>
-        <ProjectContent title="Göker Blog" usedTechnology="React, Css, HTML" summery="summurryfvfv" projectlink="/day1"></ProjectContent>
-        <ProjectContent title="React: Very First Steps" usedTechnology="React" summery="summurryfvfv" projectlink="/day1"></ProjectContent>
-        <ProjectContent title="React: Very First Steps" usedTechnology="React" summery="summurryfvfv" projectlink="/day1"></ProjectContent>
-        <ProjectContent title="React: Very First Steps" usedTechnology="React" summery="summurryfvfv" projectlink="/day1"></ProjectContent>
-
+        <PageHeader> My projects </PageHeader>
+        {datas.map(data => (
+            <ProjectContent
+                title={data.title}
```

Comment:
There must be a key here for react to optimize the rendering. please add and id to the projectDatas each item. These ids must be unique in the list and user this id as `key={data.id}`